doc/DeveloperGuide.md

```diff
@@ -5,16 +5,21 @@
 **Prerequisites**
 
 * JDK 8 or later 
-* Eclipse IDE
+* IntelliJ IDE
 
-**Importing the project into Eclipse**
+**Importing the project into IntelliJ**
 
 0. Fork this repo, and clone the fork to your computer
-1. Open Eclipse
-2. Click `File` > `Import`
-3. Click `General` > `Existing Projects into Workspace` > `Next`
-4. Click `Browse`, then locate the project's directory
-5. Click `Finish`
+1. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
+2. Set up the correct JDK version
+   1. Click `Configure` > `Project Defaults` > `Project Structure`
+   2. Click `New...` and find the directory of the JDK
+3. Click `Import Project`
+4. Locate the project directory and click `OK`
+5. Select `Create project from existing sources` and click `Next`
+6. Rename the project if you want. Click `Next`
+7. Ensure that your src folder is checked. Keep clicking `Next`
```

Comment:
There are two folders in the list on my screen. Do I need to check both `\src` and `\test\java`?